doc/Publishing.md

```diff
@@ -12,7 +12,7 @@ Before publishing it's important to make sure that theia-apps builds against 'ne
 
 - Make sure that there is no pending build on Theia master, otherwise a new "next" version might be published while we validate the current "next".
 
-- Go in the theia-apps repo [here](https://github.com/theia-ide/theia-apps/commits/master) and identify the latest commit. There should be an icon next to it; either a red X or a green checkmark. Click on it to go the the travis page. There re-trigger the build. We need to make sure that at least the various "next" builds passes. If it doesn't, it needs to be fixed before continuying.
+- Go in the theia-apps repo [here](https://github.com/theia-ide/theia-apps/commits/master) and identify the latest commit. There should be an icon next to it; either a red X or a green checkmark. Click on it to go the the travis page. There re-trigger the build. We need to make sure that at least the various "next" builds passes. If it doesn't, it needs to be fixed before continuing.
```

Comment:
- `the the travis page` -> the Travis page.
 - at least the various "next" builds passes -> at least one of the various... (Just asking though)

Reply:
@kittaakos I think all the "next" CI builds ought to pass. Each Docker image uses different extensions, so if there is an issue with one particular extension, images that do not use it could (in theory) pass CI. e.g. the pretty basic "theia-docker" image might still pass CI even if there is an issue with the go extension.

Reply:
I see. Then _"next" builds pass_ and not _passes_, right? Sorry for being nitpicky, I do not understand the requirements.

Reply:
Agreed